cmd/keytransparency-signer/backend.go

```diff
@@ -45,7 +45,8 @@ import (
 var (
 	serverDBPath  = flag.String("db", "db", "Database connection string")
 	domain        = flag.String("domain", "example.com", "Distinguished name for this key server")
-	epochDuration = flag.Duration("period", time.Second*60, "Time between epoch creation")
+	minEpochDuration = flag.Duration("min-period", time.Second*60, "Minimum time between epoch creation (create epochs only if there where mutations)")
+	maxElapsedEpochs = flag.Uint("max-period", 60, "Maximum elapsed min-periods until epoch creation (independent from mutations)")
```

Comment:
Done (in https://github.com/google/keytransparency/pull/593/commits/08bac3f7af45be342be3a6b758f949e5d4aebefe)